Tools/i18n/pygettext.py

```diff
@@ -618,10 +636,10 @@ def write_pot_file(messages, options, fp):
             # to skip translating some unimportant docstrings.
             print('#, docstring', file=fp)
         if msg.msgctxt is not None:
-            print('msgctxt', normalize(msg.msgctxt, encoding), file=fp)
-        print('msgid', normalize(msg.msgid, encoding), file=fp)
+            print('msgctxt', normalize(msg.msgctxt, encoding, options), file=fp)
```

Comment:
You need to pass also the length of the prefix to not overflow the single not line.